### ISSUE TITLE
oauth2client open browser window

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#84]
```
Hi. I like this stuff

http://code.google.com/p/google-api-python-client/wiki/OAuth2

but I don't love "Go to the following link in your browser:". Can I help my 
user by using Python's `webbrowser.open` to the open the URL for them (as well 
as printing it). How can I do this?

http://docs.python.org/library/webbrowser.html
```

Original issue reported on code.google.com by `matt.hic...@gmail.com` on 10 Jan 2012 at 7:10

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e56e7d073a8f.
```

Original comment by `jcgregorio@google.com` on 17 Jan 2012 at 2:14
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Nice work. Thanks JC
```

Original comment by `matt.hic...@gmail.com` on 17 Jan 2012 at 4:34
